conf/ansible/inventory/group_vars/defaults.yaml

```diff
@@ -64,6 +64,8 @@ aem:
     version: 4.3.1
   artifacts_base: overwrite-me
   enable_custom_image_provisioner: false
+  aem_license_sm_parameter: overwrite-me
```

Comment:
Also, why is this parameter under `aem` section when the relevancy is specific to `aws`?
Shouldn't this be under `aws` ?

Reply:
It will be moved to the aws section.